Workspace/src/ping_test/src/ping_client.cpp

```diff
@@ -0,0 +1,162 @@
+#include "ping_test.h"
```

Comment:
Is this all done in multiple nodes or can we make different objects to pass signals around?